core/src/main/java/org/infinispan/xsite/XSiteAdminOperations.java

```diff
@@ -187,11 +187,13 @@ public String takeSiteOffline(@Parameter(name = "site", description = "The name
          return incorrectSiteName(site);
       }
 
+      // TODO [pruivo] TakeOfflineManager is cluster-wide and this command is not longer required.
+      // It is kept for backwards compatible reasons and can be removed after the product release.
```

Comment:
Is this necessary given that we don't support xsite with different Infinispan versions? The scope for zero-downtime Keycloak currently does not include xsite either.

Reply:
These commands are local to the cluster. I would rather be safe than sorry and remove this after the product release.

Reply:
Ok. Please track this in a Jira